[Doc] Live Queries dashboard (#10090)
opensearch-project/documentation-website: _observing-your-data/query-insights/query-insights-dashboard.md

- When setting data retention periods, consider shorter retention periods that save storage but reduce the number of long-term insights.
- Enable metrics based on your monitoring needs. Monitoring fewer metrics prevents system overload.

## Live queries

> The **Live queries** page provides real-time visibility into search queries currently running in your OpenSearch cluster. It enables active monitoring, fast debugging, and insight into how a query;s load is distributed across nodes and indexes.

Reviewer: ```suggestion
The **Live queries** page provides real-time visibility into search queries currently running in your OpenSearch cluster. It enables active monitoring, fast debugging, and insight into how a query's load is distributed across nodes and indexes.
```

The following image shows the live queries view:

![Live Queries Dashboard]({{site.url}}{{site.baseurl}}/images/Query-Insights/Live_Queries.png)

### Metrics overview

> The top panel in live queiries view displays the following key real-time metrics:

Reviewer: ```suggestion
The top panel in live queries view displays the following key real-time metrics:
```

| **Total memory usage**   | The total memory consumed by live queries.                                      |

### Breakdown charts

Two visual charts provide breakdowns of query load:

- **By node** – Shows how many queries are running on each node.
- **By index** – Displays how many queries are targeting each index.

![Live Queries visual charts]({{site.url}}{{site.baseurl}}/images/Query-Insights/Live_queries_visuailization.png)

You can toggle between **Donut** and **Bar** chart formats using the chart type switch.

Only the top-9 items are displayed individually in the chart; additional values are grouped under the **Others** category.

### Live query table

The live query table lists the following information for each live query:

![Live Queries Table]({{site.url}}{{site.baseurl}}/images/Query-Insights/Live_Queries_Table.png)

| Column  | Description |
| :--- | :--- |
| **Timestamp**        | The time when the query started running.                            |
| **Task ID**          | The unique identifier for the search task.                            |
| **Index**            | The index or indexes the query is targeting.                          |

> | **Node**             | The dode currently running the query.                               |

Reviewer: ```suggestion
| **Node**             | The node currently running the query.                               |
```

> | **Index**            | Index or indices the query is targeting.                          |

Reviewer: ```suggestion
| **Index**            | The Index or indexes the query is targeting.                          |
```

| **Time elapsed**     | The execution time for the query so far (formatted).                                |
| **CPU usage**        | The cumulative CPU time used by the query.                            |
| **Memory usage**     | The amount of memory consumed by the query so far.                              |
| **Search type**      | The search execution method, such as `query_then_fetch`.               |

> | **Coordinator node** | The dode that coordinated the query.                                  |

Reviewer: ```suggestion
| **Coordinator node** | The node that coordinated the query.                                  |
```
